app/controllers/dogs_controller.rb

```diff
@@ -60,10 +60,15 @@ class DogsController < ApplicationController
   before_action :edit_dog_check, only: %i(edit update)
 
   def index
-    is_manager = signed_in? && session[:mgr_view]
+    is_manager = signed_in?
```

Comment:
this is a little strange, search functionality changes based on mgr_view, should be ok to remove yeah?